src/drv_oculus_rift/rift.c

```diff
@@ -449,14 +1093,55 @@ static void get_device_list(ohmd_driver* driver, ohmd_device_list* list)
 
 				desc->revision = rd[i].rev;
 		
-				desc->device_class = OHMD_DEVICE_CLASS_HMD;
+				desc->device_class = OHMD_DEVICE_CLASS_HMD | OHMD_DEVICE_CLASS_CONTROLLER;
```

Comment:
Issue #233 needs resolving before this gets pushed. If device_class isn't supposed to be a bitmask, I'll need to publish the remote buttons as a separate controller

Reply:
I think this should not be a bitfield, and HMD class devices should be allowed to register controls anyway (for example for the volume buttons on PSVR or the system button on the Vive).

Reply:
Removed the controller from the device_class, and left the Oculus remote controls published on the HMD device